Plugins/nosTrack/Track.noscfg

```diff
@@ -1,30 +1,36 @@
 {
-	"info": {
-		"id": {
-			"name": "nos.track",
-			"version": "1.7.0"
-		},
-		"display_name": "Track",
-		"category": "Virtual Studio"
-	},
-	"node_definitions": [
-		"Config/FreeD.nosdef",
-		"Config/UserTrack.nosdef"
-	],
-	"custom_types": [],
-	"binary_path": "Binaries/nosTrack",
-	"associated_nodes": [
-		{
-			"category": "nosTrack",
-			"class_name": "FreeD",
-			"display_name": "FreeD",
-			"name_aliases": ["virtual camera", "udp port"]
-		},
-		{
-			"category": "nosTrack",
-			"class_name": "UserTrack",
-			"display_name": "User Track",
-			"name_aliases": ["virtual camera"]
-		}
-	]
+  "info": {
+    "id": {
+      "name": "nos.track",
+      "version": "1.8.0"
+    },
+    "display_name": "Track",
+    "category": "Virtual Studio",
+    "dependencies": [
+      {
+        "name": "nos.sys.animation",
```

Comment:
Note: When/if optional dependencies are supported by Nodos, nos.track should not hardly dependent on `nos.sys.animation`.